docs/userguide/data_buildtool.md

```diff
@@ -0,0 +1,380 @@
+
+#### Locale-Tree Categories
+
+Several categories have the `_tree` suffix.  These categories are for "locale trees": they contain locale-specific data.  ***The [localeFilter configuration option](#slicing-data-by-locale) sets the default file filter for all `_tree` categories.***
+
+If you want to include different locales for different locale file trees, you can override their filter in the *featureFilters* section of the config file.  For example, to include only Italian data for currency symbols *instead of* the common locales specified in *localeFilter*, you can do the following:
+
+    featureFilters:
+      curr_tree: {
+        filterType: locale
+        whitelist: [
+          it
+        ]
+      }
+    }
+
+It may be common to simply exclude an entire `_tree` category without affecting other categories.  For example, to exclude region display names:
+
+    featureFilters: {
+      region_tree: {
+        filterType: exclude
+      }
+    }
+
+Note that you are able to use any of the other filter types for `_tree` categories, but you must be very careful that you are including all of the correct files.  For example, `en_GB` requires `en_001`, and you must always include `root`.  If you use the "language" or "locale" filter types, this logic is done for you.
+
+### Resource Bundle Slicing (fine-grained features)
+
+The third section of the ICU filter config file is *resourceFilters*.  With this section, you can dive inside resource bundle files to remove even more data.
+
+You can apply resource filters to all locale tree categories as well as to categories that include resource bundles, such as the `"misc"` category.
+
+For example, consider measurement units.  There is one unit file per locale (example: [en.txt](https://github.com/unicode-org/icu/blob/master/icu4c/source/data/unit/en.txt)), and that file contains data for all measurement units in CLDR.  However, if you are only formatting distances, for example, you may need the data for only a small set of units.
+
+Here is how you could include units of length in the "short" style but no other units:
+
+    resourceFilters: [
+      {
+        categories: [
+          unit_tree
+        ]
+        rules: [
+          -/units
+          -/unitsNarrow
+          -/unitsShort
+          +/unitsShort/length
+        ]
+      }
+    ]
+
+Conceptually, the rules are applied from top to bottom.  First, all data for all three styes of units are removed, and then the short length units are added back.
+
+#### Wildcard Character
+
+You can use the wildcard character (`*`) to match a piece of the resource path.  For example, to include length units for all three styles, you can do:
+
+    resourceFilters: [
+      {
+        categories: [
+          unit_tree
+        ]
+        rules: [
+          -/units
+          -/unitsNarrow
+          -/unitsShort
+          +/*/length
+        ]
+      }
+    ]
+
+At this time, the wildcard character must stand alone in a piece of the path.
```

Comment:
Reading this, I was confused whether "At this time" referred to the flow of this document.

How about "The wildcard must be the only character in its path segment. Future ICU versions may expand the syntax."

Reply:
Thanks; done